lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -384,26 +382,28 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                   <path d="M19 5h-2V3H7v2H5c-1.1 0-2 .9-2 2v1c0 2.55 1.92 4.63 4.39 4.94A5.01 5.01 0 0011 15.9V19H7v2h10v-2h-4v-3.1a5.01 5.01 0 003.61-2.96C19.08 12.63 21 10.55 21 8V7c0-1.1-.9-2-2-2zM5 8V7h2v3.82C5.84 10.4 5 9.3 5 8zm14 0c0 1.3-.84 2.4-2 2.82V7h2v1z" fill="currentColor"></path>
                 </svg>
                 <div class="message-copy" align="auto">
-                % if fbe_is_on:
-                  ${Text(_("{start_bold}Get the most out of your course!{end_bold} Upgrade to get full access to the course material, unlock both graded and non-graded assignments, and earn a {link_start}verified certificate{link_end} to showcase on your resumé.")).format(
-                          start_bold=HTML('<strong class="message-copy-bold">'),
-                          end_bold=HTML('</strong>'),
-                          link_start=HTML('<a href="{}" class="verified-info" data-course-key="{}">').format(
-                            marketing_link('WHAT_IS_VERIFIED_CERT'),
-                            enrollment.course_id
-                          ),
-                          link_end=HTML('</a>')
-                        )}
-                % else:
-                  ${Text(_("{start_bold}Get the most out of your course!{end_bold} Upgrade to earn a {link_start}verified certificate{link_end} to showcase on your resumé.")).format(
-                          start_bold=HTML('<strong class="message-copy-bold">'),
-                          end_bold=HTML('</strong>'),
-                          link_start=HTML('<a href="{}" class="verified-info" data-course-key="{}">').format(
-                            marketing_link('WHAT_IS_VERIFIED_CERT'),
-                            enrollment.course_id
-                          ),
-                          link_end=HTML('</a>')
-                        )}
+                % if fbe_status_list:
+                  % if fbe_status_list[dashboard_index]:
```

Comment:
For optional consideration (I'm approving either way): 
Without that previous `fbe_is_on` variable name around, I momentarily forgot what's in that list, and I had to look back to remember- it's got an fbe boolean for each enrollment. That might just be me, but I thought I'd mention in case you prefer it too: a slightly more foolproof name for that list could be `enrollments_fbe_status`.